lib/spring/client/run.rb

```diff
@@ -1,3 +1,4 @@
+require "spring/commands"
```

Comment:
I had to require `spring/command` cause it [loads the user defined config files](https://github.com/rails/spring/blob/master/lib/spring/commands.rb#L28-L29) (`config/spring.rb` or `~/.spring.rb`).

If user has defined `Spring.quite` on one of those files then we need different behavior [here](https://github.com/rails/spring/pull/455/files#diff-c1a62c31f3c02a71c2d99bf2faee9457R142):

```
puts "Running via Spring preloader in process #{pid}" unless Spring.quiet
```